connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SourceConnectorConfig.java

```diff
@@ -129,7 +129,8 @@ public SourceConnectorConfig(Plugins plugins, Map<String, String> props, boolean
             propsWithoutRegexForDefaultGroup.entrySet()
                     .removeIf(e -> e.getKey().equals(DEFAULT_TOPIC_CREATION_PREFIX + INCLUDE_REGEX_CONFIG)
                             || e.getKey().equals(DEFAULT_TOPIC_CREATION_PREFIX + EXCLUDE_REGEX_CONFIG));
-            enrichedSourceConfig = new EnrichedSourceConnectorConfig(enrich(defaultConfigDef, props,
+            enrichedSourceConfig = new EnrichedSourceConnectorConfig(plugins,
+                    enrich(defaultConfigDef, props,
                     defaultGroup), propsWithoutRegexForDefaultGroup);
```

Comment:
Super tiny nit: the wrapping even from the original was confusing, and maybe it's worth changing one more line and cleaning up the wrapping:
```suggestion
            enrichedSourceConfig = new EnrichedSourceConnectorConfig(plugins,
                    enrich(defaultConfigDef, props, defaultGroup), 
                    propsWithoutRegexForDefaultGroup);
```
WDYT?

Reply:
Good catch. The break was added by accident there.